php-classes/Slate/CBL/Tasks/TasksRequestHandler.php

```diff
@@ -190,4 +197,11 @@ public static function getArchivedConditions($includeArchived) {
             }
         }
     }
+
+    public static function getSharedConditions($includeShared) {
+        $recordClass = static::$recordClass;
+        if (!$includeShared) {
+            return $recordClass::getTableAlias().'.Status != "shared"';
+        }
+    }
```

Comment:
```suggestion
```

Reply:
this function is no longer in use and has been removed